system-variables.md

```diff
@@ -844,7 +827,7 @@ Constraint checking is always performed in place for pessimistic transactions (d
 
 ### tidb_enable_stmt_summary <span class="version-mark">New in v3.0.4</span>
 
-- Scope: SESSION | GLOBAL
+- Scope: GLOBAL
```

Comment:
This behavior change is intentional and only affects master. See: https://github.com/pingcap/tidb/blob/master/docs/design/2021-12-08-instance-scope.md